### PR TITLE
More updates related to mapdata fix

### PR DESCRIPTION
**Pull request by poiuyqwert**
- Update map 0 data name to "No map"
- Fix mapdata.tbl offset in DATEntryName
- Update test_entry_names for mapdata fix

### CODE REVIEW

**Comment by poiuyqwert** [comment]
Hey @neivv, are you able to do a quick sanity check on this? This slight off by 1 in multiple places just really confuses me lol. Thank you 🙏

File: PyMS/Tests/DAT/test_entry_name.py
Comment (neivv) [inline PyMS/Tests/DAT/test_entry_name.py:1]: Comment in this file, at line 1196 is now incorrect too.

**Comment by neivv** [comment]
Yea these are now correct, sorry for not noticing that you had tests for these.

Also, not exactly mapdata changes, but the `offset` parameter on `DATEntryName._entry_label` seems useless? Only user left now is portdata and there it also appears to be off by one? With the way strings in portdata.tbl are laid out, with default files it just gets talking portrait name instead idle name, but nevertheless.

File: PyMS/Tests/DAT/test_entry_name.py
Comment (poiuyqwert) [reply PyMS/Tests/DAT/test_entry_name.py:1]: Nice catch, thanks!

**Comment by poiuyqwert** [comment]
Good catch, there were even some other issues with the portrait tests caused by that. Should all be fixed now.

**Pull request merged**